x-pack/plugins/apm/server/routes/mobile/get_mobile_crash_rate.ts

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ProcessorEvent } from '@kbn/observability-plugin/common';
+import {
+  kqlQuery,
+  rangeQuery,
+  termQuery,
+} from '@kbn/observability-plugin/server';
+import { Coordinate } from '../../../typings/timeseries';
+import { Maybe } from '../../../typings/common';
+import { APMEventClient } from '../../lib/helpers/create_es_client/create_apm_event_client';
+import { getBucketSize } from '../../../common/utils/get_bucket_size';
+import {
+  ERROR_TYPE,
+  ERROR_ID,
+  SERVICE_NAME,
+} from '../../../common/es_fields/apm';
+import { environmentQuery } from '../../../common/utils/environment_query';
+import { getOffsetInMs } from '../../../common/utils/get_offset_in_ms';
+import { offsetPreviousPeriodCoordinates } from '../../../common/utils/offset_previous_period_coordinate';
+
+export interface CrashRateTimeseries {
+  currentPeriod: { timeseries: Coordinate[]; value: Maybe<number> };
+  previousPeriod: { timeseries: Coordinate[]; value: Maybe<number> };
+}
+
+interface Props {
+  apmEventClient: APMEventClient;
+  serviceName: string;
+  transactionName?: string;
+  environment: string;
+  start: number;
+  end: number;
+  kuery: string;
+  offset?: string;
+}
+
+async function getMobileCrashTimeseries({
+  apmEventClient,
+  serviceName,
+  transactionName,
+  environment,
+  start,
+  end,
+  kuery,
+  offset,
+}: Props) {
+  const { startWithOffset, endWithOffset } = getOffsetInMs({
+    start,
+    end,
+    offset,
+  });
+
+  const { intervalString } = getBucketSize({
+    start: startWithOffset,
+    end: endWithOffset,
+    minBucketSize: 60,
+  });
+
+  const aggs = {
+    crashes: {
+      cardinality: { field: ERROR_ID },
+    },
+  };
+
+  const response = await apmEventClient.search('get_mobile_crash_rate', {
+    apm: {
+      events: [ProcessorEvent.error],
+    },
+    body: {
+      track_total_hits: false,
+      size: 0,
+      query: {
+        bool: {
+          filter: [
+            ...termQuery(ERROR_TYPE, 'crash'),
+            ...termQuery(SERVICE_NAME, serviceName),
+            ...rangeQuery(startWithOffset, endWithOffset),
+            ...environmentQuery(environment),
+            ...kqlQuery(kuery),
+          ],
+        },
+      },
+      aggs: {
+        timeseries: {
+          date_histogram: {
+            field: '@timestamp',
+            fixed_interval: intervalString,
+            min_doc_count: 0,
+            extended_bounds: { min: startWithOffset, max: endWithOffset },
+          },
+          aggs,
+        },
+        ...aggs,
+      },
+    },
+  });
+
+  const timeseries =
+    response?.aggregations?.timeseries.buckets.map((bucket) => {
+      return {
+        x: bucket.key,
+        y: bucket.crashes.value,
+      };
```

Comment:
Here we are returning the number of crashes but not the crash rate, we calculate the crash rate getting the crashes per session, or at least it's how we did it for the tables on transactions view. 
Did something change?
https://github.com/elastic/kibana/blob/f1402d682c6aef8486fdcb465b9770afb93b79b7/x-pack/plugins/apm/server/routes/mobile/get_mobile_main_statistics_by_field.ts#L199